pkg/middlewares/replicate/replicate.go

```diff
@@ -118,6 +129,12 @@ func (r *replicate) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		}
 	}
 
+	size := len(eventRequest.Body) + len(eventResponse.Body)
+	if size > r.maxProcessableBodySize {
+		logger.Debugf("ignoring request and result with too long body: total length is %d", size)
```

Comment:
```suggestion
		logger.Debugf("ignoring request and result with too long body: total length is %d", size)
```
		logger.Debugf("ignoring request and response with too long body: total length is %d", size)